src/test/java/uk/gov/digital/ho/hocs/audit/export/AuditExportServiceTest.java

```diff
@@ -466,6 +512,16 @@ public void verifyHeadersAreSubstitutedWithCorrespondentExtract() throws IOExcep
         verify(passThroughHeaderConverter, times(1)).substitute(anyList());
     }
 
+    @Test
+    public void verifyHeadersAreSubstitutedWithExtractExtract() throws IOException {
```

Comment:
Should the method name on this test be `ExtensionExtract`?

Reply:
I have addressed this